File: Example C# Sharp/Example/Select_Form.cs
using System;
using System.Windows.Forms;

namespace Example
{
    public partial class Select_Form : Form
    {
        public Select_Form()
        {
            InitializeComponent();
        }

        // Button click handler for License authentication option
        private void button1_Click(object sender, EventArgs e)
        {
            // Hide the current selection form
            this.Hide();
            
            // Create and show the License test form
            Test_License testLicenseForm = new Test_License();
            testLicenseForm.Show();
        }

        // Button click handler for Account authentication option
        private void button2_Click(object sender, EventArgs e)
        {
            // Hide the current selection form
            this.Hide();
            
            // Create and show the Account test form
            Test_Account testAccountForm = new Test_Account();
            testAccountForm.Show();
        }
    }
}

File: Example VB.NET/Example/Select_Form.vb
Public Class Select_Form
    ''' <summary>
    ''' Event handler for Account authentication button
    ''' Opens the username/password authentication form
    ''' </summary>
    Private Sub Button_Account_Click(sender As Object, e As EventArgs) Handles Button_Account.Click
        ' Show the account authentication form
        Test_Account.Show()
        ' Hide the current selection form
        Me.Hide()
    End Sub

    ''' <summary>
    ''' Event handler for License Key authentication button
    ''' Opens the license key authentication form
    ''' </summary>
    Private Sub Button_Key_Click(sender As Object, e As EventArgs) Handles Button_Key.Click
        ' Show the license key authentication form
        Test_License.Show()
        ' Hide the current selection form
        Me.Hide()
    End Sub
End Class